charts/kcp/templates/kcp.yaml

```diff
@@ -323,6 +326,10 @@ spec:
           mountPath: /etc/kcp/tls/requestheader-client
         - name: kubeconfig
           mountPath: /etc/kcp/config
+        {{- if .Values.kcp.tokenAuth.enabled }}
+        - name: kcp-token-auth-file
+          mountPath: /etc/kcp/token-auth
+        {{- end}}
```

Comment:
nit - would be nice to break the change into logically independent commits, e.g enabling token auth isn't strictly related to the KIND helper scripts